contrib/mesos/pkg/queue/historical.go

```diff
@@ -377,6 +377,16 @@ func (f *HistoricalFIFO) merge(id string, obj UniqueCopyable) (notifications []E
 	return
 }
 
+// Resync will touch all objects to put them into the processing queue
+func (f *HistoricalFIFO) Resync() error {
+	for _, entry := range f.items {
+		if err := f.Update(entry); err != nil {
```

Comment:
thanks for catching this, the implementation is very wrong. `items` contains records (of type `entry`) of present and past events. should only be re-queuing objects for ADD- or UPDATE- type entries.